README.md

```diff
@@ -3,11 +3,11 @@
 
 ![Genesys screenshot](genesys-screenshot.png)
 
-Genesys provides a simple way to make rich, accessible, interactive prototypes using a declarative language (HTML). It enables designers and UI developers to quickly create low to high-fidelity prototypes, or even production-ready UI. 
+Genesys provides a simple way to make rich, accessible, interactive prototypes using a declarative language (HTML). It enables designers and UI developers to quickly create a low to high-fidelity prototypes, or even production-ready User Interface. 
 
-Genesys provides support for most major design systems (Bootstrap, Material Design, Foundation, Ionic, Tailwind, Tachyons, et. al.) and you can add any others. 
+Genesys provides support for most major design systems like Bootstrap, Material Design, Foundation, Ionic, Tailwind, Tachyons, et. al.) and you can add many others. 
```

Comment:
I prefer the use of 'any' versus 'many', since 'many' implies limitations on what design systems can be added, and Genesys does not have that limitation.

Reply:
Yes I Agree with you